pages/dkp/kommander/2.1/install/air-gapped/index.md

```diff
@@ -26,6 +28,52 @@ Before installing, ensure you have:
 
 - [MetalLB enabled and configured][air-gap-install-metallb], which provides load-balancing services.
 
+### Kommander charts bundle
+
+Charts bundle is a Gziped Tar archive containing Helm charts required during Kommander installation.
+It can be created using Kommander CLI or downloaded together with Kommander CLI
```

Comment:
Create the charts bundle with the Kommander CLI or download along with the Kommander CLI.